README.md

```diff
@@ -0,0 +1,29 @@
+## theme-example for ownCloud
+
+### Use the theme-example App
+To use the theme-example app, clone or download this repository to the `owncloud\apps` folder on your server. Please note that the Apache user needs access rights to the folder and its content. As an example, use following command to adjust the permissions: `chmod -R www-data:www-data theme-example`.
+
+If you change the name of the app from `theme-example` to another name, be sure to change the name in the file `appinfo\info.xml` as well. The folder name and the `<id>` must be identical.
+
+### Edit images
+To edit images it is best to open the original images from the 'core\img` folder with Gimp or Photoshop. After you have made changes, overwrite the originals.
```

Comment:
The final correction: you have to change the leading apostroph to backtick in core\img.
For highlighting, you need to have a leading and trailing backtick. Else the render enginge confuses.
This must have been slipped thru...